homeworks/03-data-structure/tests/BinaryTree.test.ts

```diff
@@ -0,0 +1,139 @@
+import { BinaryTree, TraverseType } from '../src/BinaryTree';
+
+describe('BinaryTree', () => {
+  describe('left', () => {
+    it('Null if no Left Branch', () => {
```

Comment:
`Null` and undefined is not the same -> `Undefined if no Left Branch` would be more appropriate test title.